classes/Kohana/Log/File.php

```diff
@@ -84,7 +84,9 @@ public function write(array $messages)
 			chmod($filename, 0666);
 		}
 
-		foreach ($messages as $message)
+		$filtered_messages = $this->filter($messages);
+
+		foreach ($filtered_messages as $message)
```

Comment:
I might have missed it, but I don't think the filtering within the log writers (any of them) is covered by tests?